checker/tests/stubparser-nullness/NoExplicitAnnotations.java

```diff
@@ -51,6 +51,7 @@ void use(
             NoExplicitAnnotationsSub2 sub2,
             NoExplicitAnnotationsSub3 sub3) {
         nble = sub1.method1();
+        // :: error: (assignment.type.incompatible)
```

Comment:
Can you provide a reason why this new error is desirable? Why is this change in behavior an improvement?

Reply:
Also note that a wilcard jtreg test fails.

Reply:
> Can you provide a reason why this new error is desirable? Why is this change in behavior an improvement?

Because `nn` is @Nonnull and method1() is `@Nullable`. So there should be an assignment incompatiable error.

Reply:
But `method1` is in the stub file without annotations.
Shouldn't the logic be that, if something appears in the stub-file, those annotations are used. If there are no annotations in the stub-file, the usual defaults are used.
That's at least what the current behavior seems to be. This makes the stub-files readable and understandable without looking at what the corresponding source says.